easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb

```diff
@@ -0,0 +1,42 @@
+name = 'TensorFlow'
+version = '1.7.0'
+cuda_ver = '9.1.85'
+versionsuffix = '-Python-%%(pyver)s-CUDA-%s' % cuda_ver
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-1.5.0_swig-env.patch',
+    'TensorFlow-1.6.0_dont_expand_cuda_cudnn_path.patch',
+    'TensorFlow-1.6.0_remove-msse-hardcoding.patch',
+    'TensorFlow-1.6.0_lrt-flag.patch',
+]
+checksums = [
+    'c676a96fc8700722816b2b98c85578b2f99fac7a7b2484c9c7f0641484f8d50d',  # v1.7.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.5.0_swig-env.patch
+    # TensorFlow-1.6.0_dont_expand_cuda_cudnn_path.patch
+    'f490b35179ed5c864255950784d5677d02f6c83a1684d406dbd31e2607d342fa',
+    '0ad82aab981721b1ed4881106d9accd43fe38b863ec147f6a16b278605012099',  # TensorFlow-1.6.0_remove-msse-hardcoding.patch
+    'd8b262eea9ff94c94c90b8c51219d8f56085ccee4370525cce8beba3b2b91d38',  # TensorFlow-1.6.0_lrt-flag.patch
+]
+
+builddependencies = [
+    ('Bazel', '0.11.1'),
+    ('wheel', '0.30.0', '-Python-%(pyver)s'),
+]
+dependencies = [
+    ('Python', '3.6.4'),
+    ('cuDNN', '7.0.5', '-CUDA-%s' % cuda_ver, True),
+]
+
+with_mkl_dnn = False
```

Comment:
@smoors Why is this required? Maybe clarify this with a comment above this line?

Reply:
don't know, I just copied from the previous version. maybe @wpoely86 knows more?

Reply:
it can be dropped. The easyblock should handle it.